src/styles/main.scss

```diff
@@ -1,3 +1,73 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+$month-length: "28" 28, "29" 29, "30" 30;
```

Comment:
It's better to use a list instead of a map for `$month-length`, e.g. `(28, 29, 30, 31)`